test/object_store/test_backup.py

```diff
@@ -271,7 +271,12 @@ def list_sstables():
     print('Try to restore')
     tid = await manager.api.restore(server.ip_addr, ks, cf, s3_server.address, s3_server.bucket_name, prefix, toc_names)
     status = await manager.api.wait_task(server.ip_addr, tid)
-    assert (status is not None) and (status['state'] == 'done')
+    assert status is not None
+    assert status['state'] == 'done'
+    assert status['progress_units'] == "sstables"
+    assert status['progress_completed'] == len(toc_names)
+    assert status['progress_total'] == len(toc_names)
```

Comment:
Hm... Taking into account progress from "fully contained" sstables is ignored, how does this pass? Is this test run without tablets enabled?

Reply:
i am not sure, probably because we don't set `enable_tablets` in `db/config.cc`?  anyway, i added a commit to set it for the `object_store` tests.